### PR TITLE
(fix) use optional chaining to safely access the current session provider

### PR DESCRIPTION
**Pull request by charity553**
## Requirements
- [ ] This PR has a title that briefly describes the work done including a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) type prefix and a Jira ticket number if applicable. See existing PR titles for inspiration.

#### For changes to apps
- [ ]  My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide) and [**design documentation**](https://zeroheight.com/23a080e38/p/880723-introduction).

#### If applicable
- [ ] My work includes tests or is validated by existing tests.

## Related Issue

https://openmrs.atlassian.net/browse/O3-3528

### CODE REVIEW

**Review by donaldkibet — APPROVED** [review]

File: src/form/radiology-orders/radiology-order-basket-panel/radiology-order-basket-panel.scss [hunk 1]
```diff
@@ -3,16 +3,16 @@
 @import '@openmrs/esm-styleguide/src/vars';
 
 .desktopTile {
-  border-left: 4px solid #6929c4;
+  border-left: 4px solid #18cd69;
```
Comment (donaldkibet) [inline src/form/radiology-orders/radiology-order-basket-panel/radiology-order-basket-panel.scss:6]:
```suggestion
  border-left: 4px solid colors.$color-o1;
```
We should use carbon color tokens

**Review by donaldkibet — CHANGES_REQUESTED** [review]

**Comment by ojwanganto** [comment]
@charity553 - please help address the review comments

**Comment by donaldkibet** [comment]
@charity553 please fix the conflict to move this forward

**Comment by donaldkibet** [comment]
@charity553 please push the same branch, will trying to resolve the conficts i unfortunately overwrote you changes. Thanks

**Pull request closed without merging**